mobify/platform-scaffold
# Fixes missing AMP filter link redirection to PWA

Fixes missing AMP filter link redirection to PWA

**Reviewers:** @kschmidtdev 

## Changes
- Adds new product category selector
- Builds and passes in missing redirect link in `product-list-filter-modal.jsx`

## Todos
- [x] Merge develop
- [x] Test more

## How to test-drive this PR
- `cd amp && npm start`
- visit a PDP || `localhost:3000/potions.html`, click on filter and apply one
- Make sure your are redirected to the correct PWA page
